src/GafferUIBindings/PlugAdderBinding.cpp

```diff
@@ -89,13 +92,71 @@ struct PlugMenuSlotCaller
 
 };
 
+struct PlugAdderWrapper : public PlugAdder, IECorePython::Wrapper< PlugAdder >
+{
+
+	PlugAdderWrapper(PyObject *self, StandardNodeGadget::Edge edge ) : PlugAdder( edge ), IECorePython::Wrapper<PlugAdder>( self, this )
+	{
+	}
+
+	bool acceptsPlug( const Gaffer::Plug *connectionEndPoint ) const
+	{
+		IECorePython::ScopedGILLock gilLock;
+		override o = this->get_override( "acceptsPlug" );
+		if( o )
+		{
+			try
+			{
+				return o( Gaffer::PlugPtr( const_cast<Gaffer::Plug *>( connectionEndPoint ) ) );
+			}
+			catch ( error_already_set )
```

Comment:
I'm not sure it makes much difference here, but generally I think it's best to catch by const reference, so `const error_already_set &` in this case. See http://stackoverflow.com/questions/2522299/c-catch-blocks-catch-exception-by-value-or-reference for a bit of discussion.